pkg/client/s3client.go

```diff
@@ -39,12 +40,50 @@ type S3Client struct {
 	StorageClass string
 	SvcEndpoint  string
 	S3Session    *s3.S3
+	AccessKey    string
+	SecretKey    string
```

Comment:
if these aren't used anywhere else then I don't see a point in using them here!

Reply:
Removed